package.json

```diff
@@ -82,7 +82,9 @@
     "codecov": "^3.0.0",
     "coffeescript": "^2.0.3",
     "cross-env": "^5.1.1",
+    "elm": "^0.19.0",
```

Comment:
What is this dependency used for? I don't see it used anywhere in the code.

Reply:
This provides the actual `elm` binaries that `node-elm-compiler` uses to compile the source files. The package isn't a dependency of `node-elm-compiler`, so unfortunately we have to pull it in.

Reply:
Ah I see. So since this is a dev dependency in parcel, so we might need to install it in a user's project as well, right?

Reply:
Yes, that's correct.

Reply:
ok, updated to auto install `elm` when the command isn't available globally.